angularjs-1.5-basics/src/app/binding-page/example.component.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var example = {
+    bindings: {
+        obj: '=',
+        prim: '='
+    },
+    template: `
```

Comment:
__[jshint]__
* 'template literal syntax' is only available in ES6 (use 'esversion: 6').

<!-- Comment by [SideCI](https://sideci.com) -->